book/website/collaboration/hybrid-collab/hybrid-collab-challenges.md

```diff
@@ -0,0 +1,22 @@
+(cl-hybrid-collab-challenges)=
+# Challenges faced during Hybrid Collaboration
+
+### Social networking in hybrid settings
+
+In a hybrid work setting, the task of building and maintaining a local network becomes more challenging due to limited natural social interactions and a systematic imbalance between in-person and online groups. 
+It can also be a challenge for the in-person participants to simultaneously manage interactions with people online and people in the same in-person setting as the.
```

Comment:
```suggestion
It can also be a challenge for the in-person participants to simultaneously manage interactions with people online and people in the same in-person setting as them.
```